posts/2022-06-16-http-registry-testing.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: "Cargo http-registry call for testing"
+author: Arlo Siemsen
+team: The Cargo Team <https://www.rust-lang.org/governance/teams/dev-tools#cargo>
+---
+
+The Cargo [`http-registry`][http-registry] feature is ready for testing. The feature causes Cargo
+to access the crates.io index over http, rather than git. It can provide a significant
```

Comment:
Should we link to [the RFC](https://rust-lang.github.io/rfcs/2789-sparse-index.html) that defines the sparse index (HTTP) protocol here for the curious?